src/components/input.component.tsx

```diff
@@ -1,35 +1,43 @@
 import React from 'react';
-import {Text, View, TextInput, TextInputProps} from 'react-native';
-import {StyleSheet} from 'react-native';
+import {View, TextInputProps} from 'react-native';
+import styled from 'styled-components/native';
 
 type Props = TextInputProps & {
```

Comment:
como, as vezes, a gente precisa importar esse tipo em outro lugar, a gente geralmente nomeia com o nome do componente. então, aqui ficaria assim
```suggestion
// nome do componente + 'Props'
type InputComponentProps = TextInputProps & {
```
a propósito, se vc clicar uma vez em cima do `Props` e apertar f2, o vscode abre uma caixinha para você renomear esse `type` em todos os lugares que ele aparece 😱

Reply:
feito. valeu pela dica!